kodi_addon_checker/check_addon.py

```diff
@@ -64,6 +65,7 @@ def start(addon_path, repo_addons, config=None):
     addon_id = os.path.basename(os.path.normpath(addon_path))
     addon_report = Report(addon_id)
     addon_report.add(Record(INFORMATION, "Checking add-on %s" % addon_id))
+    logger = logging.getLogger(__name__)
```

Comment:
Setting this up should be the first thing we do. We can also think about miving it to a constructor, if that's possible.

Reply:
Do mean right after this `def start(addon_path, repo_addons, config=None):
` line ?

Reply:
yes